examples/todolist-react/src/hooks/useOnEnter.js

```diff
@@ -1,10 +1,13 @@
 import { useCallback } from "react";
 
-export default function useOnEnter(callback, inputs) {
-  return useCallback(event => {
-    if (event.key === "Enter") {
-      event.preventDefault();
-      callback(event);
-    }
-  }, inputs);
+export default function useOnEnter(callback) {
```

Comment:
useOnEnter should not receive another argument than the memoized callback since we only want to create it once